src/pages/EditResourcePage.tsx

```diff
@@ -108,6 +108,44 @@ const EditResourcePage: FC = () => {
 
       const responseWithCalculatedDefaults = await getResourceDefaults(startingResource.identifier);
       await saveCalculatedFields(responseWithCalculatedDefaults.data);
+      const tempResource: Resource = {
+        ...deepmerge(startingResource, responseWithCalculatedDefaults.data),
+        contributors: [
+          {
+            identifier: contributorResponse.data.identifier,
+            features: {
+              dlr_contributor_identifier: contributorResponse.data.identifier,
+              dlr_contributor_name: user.institution,
+              dlr_contributor_type: contributorFeatureNames.Institution,
+            },
+          },
+        ],
+        licenses: [emptyLicense],
+        tags: [],
+      };
+      if (
```

Comment:
foreslår å trekke ut disse store blokkene som egne funksjoner for bedre lesbarhet